### DISCUSSION TITLE
Linux install issue: lazy.lua:10: module 'lazy' not found

### DISCUSSION THREAD

**Discussion by baoyuxu** [LazyVim/LazyVim#3290] [Q&A]
I try to install lazyvim following [this](https://www.lazyvim.org/installation), and got the error below.

I'd appreciate any pointers in the general vicinity of what might be the cause or how I could deal with it further.

```
Error detected while processing /home/xxx/.config/nvim/init.lua:
E5113: Error while calling lua chunk: /home/xxx/.config/nvim/lua/config/lazy.lua:10: module 'lazy' not found:
        no field package.preload['lazy']
        no file './lazy.lua'
        no file '/__w/neovim/neovim/.deps/usr/share/luajit-2.1/lazy.lua'
        no file '/usr/local/share/lua/5.1/lazy.lua'
        no file '/usr/local/share/lua/5.1/lazy/init.lua'
        no file '/__w/neovim/neovim/.deps/usr/share/lua/5.1/lazy.lua'
        no file '/__w/neovim/neovim/.deps/usr/share/lua/5.1/lazy/init.lua'
        no file './lazy.so'
        no file '/usr/local/lib/lua/5.1/lazy.so'
        no file '/__w/neovim/neovim/.deps/usr/lib/lua/5.1/lazy.so'
        no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
        [C]: in function 'require'
        /home/xxx/.config/nvim/lua/config/lazy.lua:10: in main chunk
        [C]: in function 'require'
        /home/xxx/.config/nvim/init.lua:2: in main chunk
```

**Comment by dpetka2001** [comment]
Delete directory `~/.local/share/nvim/` directory and restart Neovim.

**Reply by baoyuxu** [reply]
Thx for your reply, but sadly I tried delete this dir and still got same error.

**Reply by dpetka2001** [reply]
Maybe there's something wrong either with your network or some gitconfig option or maybe some proxy stuff. No idea. It should install correctly by default.

PS: check that your `git` version is greater than `2.19.0`.

**Reply by rwijtvliet** [reply]
Same here. Gitversion is `2.43.0`. I'm on WSL.

**Reply by jonnjonnjo** [reply]
@rwijtvliet mine works by first deleting all files under `~/.local/share/nvim`. Make sure that you didn't delete the `/share/nvim` folder. 

and then, try to do the installation in here again http://www.lazyvim.org/installation
